code.rb

```diff
@@ -39,8 +39,7 @@
 }
 
 def morse_decoder(string)
```

Comment:
- [ ] You should create 3 methods `decode_char()` to decode a Morse code character, `decode_word()` that reuses decode_char() to decode an entire word in Morse code, and finally `decode()` method that reuses decode_word() to decode the entire message in Morse code. kindly make sure to implement the 3 methods to decode the morse code string

![Screenshot_20221215_214846](https://user-images.githubusercontent.com/25227022/207963623-a648176e-3b6c-4372-9672-e9d1253a17d1.png)